### PR TITLE
BBSR.R

### PR DESCRIPTION
**Pull request by NicoDeVeaux**
Adding a framework to test bayesianRegression.R as well as a first shot at some unit tests for it.

### CODE REVIEW

File: inferelator_ng/tests/test_bbsr_R.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+from .. import bbsr_R
+
+my_dir = os.path.dirname(__file__)
+
+class TestDR(unittest.TestCase):
+
+    def setUp(self):
+        self.brd = bbsr_R.BBSR_driver()
+        self.brd.target_directory = os.path.join(my_dir, "artifacts")
+
```
Comment (AaronWatters) [inline inferelator_ng/tests/test_bbsr_R.py:15]: To fix the travis issue add

```
        if not os.path.exists(self.brd.target_directory):
            os.makedirs(self.brd.target_directory)
```

here

Comment (NicoDeVeaux) [inline inferelator_ng/tests/test_bbsr_R.py:15]: Thanks!

**Comment by AaronWatters** [comment]
Hmmm... I'm running the tests locally and I've got 2 R processes maxing out the CPU for a couple minutes now.  Maybe we can find out more about this at 10am.

**Comment by NicoDeVeaux** [comment]
Interesting. The same things seems to be happening on Travis CI: it timed
out after 10 minutes.

But on my local, this test immediately fails due to an R dataframe access
error:

# 

ERROR: test_two_genes_nonzero_clr_two_conditions_positive_influence

## (inferelator_ng.tests.test_bbsr_R.TestDR)

Traceback (most recent call last):
  File
"/Users/ndeveaux/Dev/inferelator_ng/inferelator_ng/tests/test_bbsr_R.py",
line 132, in test_two_genes_nonzero_clr_two_conditions_positive_influence
    (betas, resc) = self.brd.run(self.X, self.Y, self.clr, self.priors)
  File "/Users/ndeveaux/Dev/inferelator_ng/inferelator_ng/bbsr_R.py", line
100, in run
    utils.call_R(driver_path)
  File "/Users/ndeveaux/Dev/inferelator_ng/inferelator_ng/utils.py", line
33, in call_R
    return subprocess.check_output(command, shell=True)
  File "//anaconda/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
CalledProcessError: Command 'R -f
/Users/ndeveaux/Dev/inferelator_ng/inferelator_ng/tests/artifacts/run_bbsr.R'
returned non-zero exit status 1

**Pull request merged**